kubernetes/linux/Dockerfile.multiarch

```diff
@@ -32,7 +32,7 @@ WORKDIR ${tmpdir}
 RUN chmod 775 $tmpdir/*.sh; sync; $tmpdir/setup.sh ${TARGETARCH}
 
 FROM ${MARINER_DISTROLESS_IMAGE} AS distroless_image
-MAINTAINER OMSContainers@microsoft.com
+LABEL maintainer="OMSContainers@microsoft.com"

```

Comment:
Why this change?

Reply:
MAINTAINER has been deprecated: https://github.com/apache/couchdb-docker/issues/126

Reply:
@wanlonghenry , we have usage of MAINTAINER in multiple places in this file. can we fix all other instances as well?

Reply:
done